src/components/header/style.module.css

```diff
@@ -185,6 +183,8 @@
 
 		& > a,
 		& > button {
+			display: flex;
+			align-items: center;
```

Comment:
Ensures the little dropdown arrow next to 'About' (or any future dropdown pages) is vertically centered:

![Screenshot of the site header. There's a triangular dropdown arrow next to the 'About' link, but it's a bit lower than center compared to the label](https://github.com/user-attachments/assets/2ea1c42e-9bcc-455c-ac11-81bd78ea3061)